include/radio_util.h

```diff
@@ -57,6 +57,10 @@ RADIO_RESP
 radio_req_resp(
     RADIO_REQ req);
 
+const char*
+radio_interface_name(
+    RADIO_INTERFACE interface);
```

Comment:
What's the purpose of this new public API? It doesn't seem to be used by the library.

Reply:
It is used by https://github.com/mer-hybris/ofono-ril-binder-plugin/pull/28.

Reply:
Then mark it with `/* Since 1.4.0 */` in both .h and .c files. Let's assume that the next version is going to be 1.4.0

Reply:
Actually, I don't think it's necessary. Let libgbiner-radio use the enum and the client of the library deal with string representations of the enum when and if it needs that.

Reply:
Ok. I dropped last two commits including the one that moves the definition of `RADIO_INTERFACE` type.